lms/db/_bulk_action.py

```diff
@@ -93,19 +93,23 @@ def upsert(self, model_class, values):
 
         if config.upsert_trigger_onupdate:
```

Comment:
I think you are right that this name could lead to confuging (are DB triggers involvced here?)

Not sure which better name we could use `upsert_orm_onupdate`? `upsert_onupdate`?

Reply:
`upsert_call_onupdate`? `upsert_use_onupdate`?

Reply:
`upsert_use_onupdate` Looks good to me